galasa-parent/dev.galasa.framework.resource.management/src/main/java/dev/galasa/framework/resource/management/internal/RunResourceManagement.java

```diff
@@ -73,6 +73,13 @@ public void start() {
         } catch (FrameworkException e) {
             logger.error("Unable to initialise Finished Run monitor", e);
         }
+        try {
+            this.resourceManagement.getScheduledExecutorService().scheduleWithFixedDelay(
+                    new RunRASMonitor(this.framework, this.resourceManagement, this.dss, cps, this),
+                    this.framework.getRandom().nextInt(20), 20, TimeUnit.SECONDS);
```

Comment:
We dont need this running every 20 seconds,   once an hour maybe

Reply:
and give it a long initial wait

Reply:
Ah that was left in from my testing